lib/mayaUsd/ufe/UsdPointInstanceModifierBase.h

```diff
@@ -275,8 +342,27 @@ class MAYAUSD_CORE_PUBLIC UsdPointInstanceModifierBase
         return usdAttr;
     }
 
+    void _closeBatch()
+    {
+        // To close the batch we simply remove it from the map of batches under
+        // construction, and thus it can no longer be joined.  If a batch was
+        // created, erase() will harmlessly fail for all modifiers except the
+        // first one.  If no batch was created, make a trivial, unshared batch
+        // just for this modifier.
+        batches().erase(pointInstancerPath());
+        if (!_batch) {
```

Comment:
When closing the batch there might not have been any created.  In that case create a trivial batch with ourselves as the single point instance in the batch.